Feat(kit-checklist): Collect the number of family members

meunomeecris opened the pull request:
- Add the brand color to .accentColor;
- Change the AppIcon;
- Create the kit d'urgence's onboarding with steps;
- Create the logic and interface for collecting the number of family members ;
- Save and load the family's member from userDefaults;
- Create unit tests for the EmergencyKitStore.


<p align="center">
<img src="https://github.com/user-attachments/assets/a5fb443d-f64e-4bd9-aa30-dc29a3fa76c9" height="300" margin-right: 10px;">
 <img src="https://github.com/user-attachments/assets/ec9730b4-7bf3-4615-a6b5-0ce132ca2d1d" height="300" margin-right: 10px;">
 <img src="https://github.com/user-attachments/assets/d936283c-5a6f-4956-8eef-4a3316b7f7e5" height="300" margin-right: 10px;">
 <img src="https://github.com/user-attachments/assets/2a45bc81-a449-472c-ae8e-ec7c68eca12f" height="300" margin-right: 10px;">
<img src="https://github.com/user-attachments/assets/8518bc62-daba-485f-ae37-42a9cc4b3573" height="300" margin-right: 10px;">
</p>

meunomeecris committed:
feat(KitDUrgence): add familyMemberView
	change the bottom bar's icon
	add AppIcon
	change "checklist" term for "emergencyKit"
	add emergencyKitStore
	add emergencyKit views

meunomeecris committed:
chore(kit-checklist): refactoring code for readability

meunomeecris committed:
chore(kit-checklist): add unit tests to EmergencyKitScore and refactoring code

meunomeecris committed:
chore(kit-checklist): save member count on userDefault

meunomeecris committed:
Update README.md

meunomeecris committed:
chore(kit-checklist): add onbardign on emergencyKit feat
	- refactoring code
	- change AppIcon

meunomeecris committed:
chore(kit-checklist): refactoring code:

meunomeecris committed:
chore(kit-checklist): apply brand colors

meunomeecris committed:
chore(kit-checklist): refactoring code
	- make better the design

meunomeecris committed:
fix(kit-checklist): unit tests

viniciusaro commented on GeoRisques/GeoRisquesTests/EmergencyKitTests.swift:
```diff
@@ -0,0 +1,27 @@
+    @Test func testIncrementCount() async throws {
+        let store = EmergencyKitStore()
+
+        store.memberCount["Adultes"] = 0
+        store.incrementMember("Adultes")
```
Se conseguirmos depois mudar essa api para utilizar o MemberType ao invés de String, é mais seguro :D OU seja, ao invés de vc ter um `memberCount: [String: Int]` precisaríamos ter um `memberCount: [MemberType: Int]`

meunomeecris replied:
Yess! I did! And also, added the protocol to save the member's infos.

viniciusaro reviewed (approved):
BOM DEMAIS AMIGA!!!!

meunomeecris committed:
chore(kit-checklist): add protocolo to save and load family's member

meunomeecris committed:
chore(kit-checklist): refactoring code
	- change name's file
	- change the type of the dictonary from String to MemberType
	- fixing code for the new dictonary type
